packages/hierarchy-builder/src/hierarchy-builder/ErrorTypeChecker.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { RowsLimitExceededError } from "./internal/TreeNodesReader";
+
+/**
+ * Namespace which can be used to determine error type.
+ * @beta
+ */
+export namespace ErrorTypeChecker {
+  /** Checks if the error is a RowsLimitExceededError. */
+  export function isRowsLimitExceededError(err: any): err is RowsLimitExceededError {
+    return err instanceof RowsLimitExceededError;
+  }
+}
```

Comment:
I'm not sure we need this... Calling `ErrorTypeChecker.isRowsLimitExceededError(error)` doesn't simplify anything compared to `error instanceof RowsLimitExceededError` call. Are there ant other reasons to have it?

Reply:
Rows limit exceeded error was in the hierarchy-builder/internal folder, and was marked with `@internal` tag. Because of this, we weren't able to import it in test-app and I assumed that I shouldn't change it.
Changed so we have `HierarchyErrors.ts` file, which exports `RowsLimitExceededError`.